packages/landing/scripts/bundle.js

```diff
@@ -99,8 +105,31 @@ async function bundleHtml(opts) {
     await fs.writeFile('./dist/index.html', htmlOutput);
 }
 
+async function recurseStatic(subDir = '') {
```

Comment:
Could we get a comment as to what this does?

This also seems to be quite similar to the logic below could they use a shared function for the work?

Reply:
DRYing up this code will involve more insertions than deletions but okay.